zipline/data/data_portal.py

```diff
@@ -554,8 +554,6 @@ def __init__(self,
                 self._equity_minute_reader,
                 self._adjustment_reader
             )
-            self.MINUTE_PRICE_ADJUSTMENT_FACTOR = \
```

Comment:
It's never used within Zipline: https://github.com/quantopian/zipline/search?utf8=%E2%9C%93&q=MINUTE_PRICE_ADJUSTMENT_FACTOR